dali/operators/image/color/brightness_contrast.h

```diff
@@ -45,9 +44,12 @@ constexpr float HalfRange() {
     : 0.5f;
 }
 
-}  // namespace detail
 }  // namespace brightness_contrast
 
+const float kDefaultBrightness = 1.f;
+const float kDefaultBrightnessShift = 0;
+const float kDefaultContrast = 1.f;
```

Comment:
Please don't. It's much easier to understand the purpose of having a multiplier of 1 or an addend of 0 than to look for definition somewhere. This also creates may create a misconception that these values are somehow arbitrarily chosen, when in fact they are mathematically fixed.